drill.cpp

```diff
@@ -852,7 +852,7 @@ map<int, drillbit> ExcellonProcessor::optimize_bits() {
             if (difference) {
                 wanted_drill_bit.diameter = best_available_drill->diameter().asInch(inputFactor);
                 wanted_drill_bit.unit = "inch";
-                if (difference > 1e-6 || difference < -1e-6) {
+                if (difference && abs(*difference) > 1e-6) {
```

Comment:
Oh man I'm so sorry to do this to you, I reviewed it on my phone and I didn't see that there is already `if(difference)` above!  I'm a dummy.

So no need to test for difference.  Doh!

Reply:
I was wondering about that, but then I don't know a lot about C++/boost :)
fixed in e8d68de